pkg/client/redhatsso/sso_auth_http_client.go

```diff
@@ -0,0 +1,25 @@
+package redhatsso
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/stackrox/acs-fleet-manager/pkg/client/iam"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+// NewSsoAuthHTTPClient returns http client which uses 2-legged OAuth2 flow to automatically get and refresh
+// access token.
+func NewSsoAuthHTTPClient(scopes []string, realmConfig *iam.IAMRealmConfig) *http.Client {
```

Comment:
```suggestion
func NewSSOAuthHTTPClient(scopes []string, realmConfig *iam.IAMRealmConfig) *http.Client {
```

Reply:
Additionally, this would also be better from consumer POV:
```suggestion
func NewSSOAuthHTTPClient(realmConfig *iam.IAMRealmConfig, scopes ...string) *http.Client {
```